assets/styles/labels.scss

```diff
@@ -51,11 +78,12 @@ $flex-align: (
     @include label;
     --label-height: 20px;
 
-    background-color: transparent;
-    border-style: solid;
-    border-width: 1px;
-    border-color: $gray-mid;
-    color: $gray-mid;
+    @each $name, $color in $label-bg-colors {
+        &.label--#{"" + $name} {
```

Comment:
imo you don't have to add that `""` there, it was an issue when table keys were actual color names